### PR TITLE
docs: add meeting minute for tapps meeting

### PR DESCRIPTION
**Pull request by varl**
Adding bullet points for the things we discussed.

Pretty sure I forgot one point we talked about, add a comment if you feel there is something missing. I'll add more information about the high-level points tomorrow, quickly writing this down before I forget something else. 🤔

### CODE REVIEW

**Comment by ismay** [comment]
> Pretty sure I forgot one point we talked about, add a comment if you feel there is something missing

I think that's everything as far as I can remember 👍

**Comment by ismay** [comment]
@varl Ah, the private-notes topic is missing

**Comment by varl** [comment]
> @varl Ah, the private-notes topic is missing

Ah, no-can-do on GitHub :neutral_face:. Our plan does not support private repos on the organisation.

File: decisions/2019/09/12-tapps-workflow-iterations.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+- Specifications (feature files) using Gherkin
+- Regression test specs for bugs, also using Gherkin to ensure that
+  fixed bugs stay out of the system
+- Does **not** replace "other" tests like unit tests
+- Does replace Zephyr test cases (need to figure out how to integrate
+  this with QA)
+- Jan-Gerke to organise a session to demonstrate how it fits together
+
+## Better communication across the entire frontend team regarding
+libraries and shared components
```
Comment (erikarenhill) [inline decisions/2019/09/12-tapps-workflow-iterations.md]: line-break makes it look bad in formatted view

File: decisions/2019/09/12-tapps-workflow-iterations.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+hour slack discussion when there are open-ended questions that needs to
+be discussed. We have been avoiding voice chats out of perceived respect
+for other's time, which is great, but sometimes the best thing is just
+to have a quick call.
+
+_Bonus: It's also nice from a team cohesion perspective to talk to
+colleagues "with voice" instead of text._
+
+## Focus on e.g. a few large refactors at a time, opting to have more
+devs working on the same thing
```
Comment (erikarenhill) [inline decisions/2019/09/12-tapps-workflow-iterations.md]: line-break makes it look bad in formatted view

File: decisions/2019/09/12-tapps-workflow-iterations.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+progress with fewer developers (often _one_). We want to shift to have a
+focus on one or two "bigger" things at any time, and more developers
+working on them at the same time:
+
+- It helps with building a cohesive code base across apps
+- Better team work
+- Spread knowledge between apps to avoid creating information silos
+
+## Manage discussions in dhis2/notes stricter to ensure one thing is
+discussed in each issue
```
Comment (erikarenhill) [inline decisions/2019/09/12-tapps-workflow-iterations.md]: line-break makes it look bad in formatted view

File: decisions/2019/09/12-tapps-workflow-iterations.md (hunk 1 above)
Comment (varl) [reply decisions/2019/09/12-tapps-workflow-iterations.md]: cefb5a6

File: decisions/2019/09/12-tapps-workflow-iterations.md (hunk 2 above)
Comment (varl) [reply decisions/2019/09/12-tapps-workflow-iterations.md]: cefb5a6

File: decisions/2019/09/12-tapps-workflow-iterations.md (hunk 3 above)
Comment (varl) [reply decisions/2019/09/12-tapps-workflow-iterations.md]: cefb5a6

**Comment by varl** [comment]
@Birkbjo is on vacation so not waiting for his approval, however, if there is anything you oppose or have a comment too, a merged PR doesn't mean that it's off limits for discussion.

**Pull request merged**